modules/trusted-firmware-m/Kconfig.psa.defconfig

```
@@ -58,7 +49,6 @@ config PSA_WANT_ALG_HMAC
 	default y
 config PSA_WANT_ALG_SHA_1
 	bool
```

This config does not apply a default value, there is no point in having it like this.

It would be better to do this:
```suggestion
	bool
	default n if BUILD_WITH_TFM
	default y
```

---

Unfortunately SHA-1 is deemed as not recommended and should be fazed out if possible according to NIST and other spec/certifying bodies. It is still in use in TLS/DTLS. Not enabling it by default and having customers enable it from samples is the best option.